docs/source/execution_device_interoperability.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<blockquote>Willing to deploy a ML service on a CPU-only server? Do the compute intensive training on GPU and run the inferences on a cheaper CPU server. </blockquote><p>CPU servers are not always cheaper; in fact for large-scale applications they are usually more expensive. Maybe switch this to something like:</p><p>While ML training workflows almost always benefit from the superior speed of GPUs, small-scale applications with limited traffic and loose latency requirements may be able to perform inference on CPU. To train a model on GPU but deploy it on CPU...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/5130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>